src/KiotaClientFactory.cs

```diff
@@ -32,17 +33,37 @@ public static HttpClient Create(HttpMessageHandler? finalHandler = null)
         /// Creates a default set of middleware to be used by the <see cref="HttpClient"/>.
         /// </summary>
         /// <returns>A list of the default handlers used by the client.</returns>
-        public static IList<DelegatingHandler> CreateDefaultHandlers()
+        public static IList<DelegatingHandler> CreateDefaultHandlers(IRequestOption[]? optionsForHandlers = null)

```

Comment:
Add this additional parameter to the Create method line 26 so it can be passed to the call.